app/components/super-features/super-features-relations/super-features-relations.html

```diff
@@ -1,7 +1,7 @@
 <div class="super-features-relations-list-header">
   <h4>Child Pages</h4>
 
-  <div ng-if="relations.length > 0">
+  <div ng-if="article.child_types.length > 0">
```

Comment:
If I'm understanding, I think the check [here](https://github.com/theonion/bulbs-cms/blob/96766b15fe5fcb2f9c6c02c875738aca08b0b65c/app/components/super-features/super-features-relations/super-features-relations.html#L18) should be sufficient to prevent creating new relations.

The check up higher here is also going to block out the `Use Parent Publish Date` button. Does this functionality need to be blocked in this case?